microservice/microservice/dbmicroservice/MicroserviceBootstrapper.cs

```diff
@@ -536,21 +526,38 @@ public static bool TryFindBeamableFolder(out string beamableFolderPath)
 			
 	        using var process = new Process();
 
+	        var dotnetPath = Environment.GetEnvironmentVariable("BEAM_DOTNET_PATH");
+	        var beamProgram = GetBeamProgram();
+
+	        string arguments = $"{beamProgram} project generate-env {serviceName} {customArgs}";
+	        string fileName = !string.IsNullOrEmpty(dotnetPath) ? dotnetPath : "dotnet";
 	        
-	        process.StartInfo.FileName = GetBeamProgram();
-	        process.StartInfo.Arguments = $"project generate-env {serviceName} {customArgs}";
+	        process.StartInfo.FileName = fileName;
+	        process.StartInfo.Arguments = arguments;
 	        process.StartInfo.RedirectStandardOutput = true;
 	        process.StartInfo.RedirectStandardError = true;
 	        process.StartInfo.CreateNoWindow = true;
 	        process.StartInfo.UseShellExecute = false;
 
+	        string path = Environment.GetEnvironmentVariable("BEAM_DOTNET_MSBUILD_PATH", EnvironmentVariableTarget.Process);
+	        if (!string.IsNullOrEmpty(path))
+	        {
+		        process.StartInfo.EnvironmentVariables["BEAM_DOTNET_MSBUILD_PATH"] = path;

```

Comment:
Create a class `EnvironmentVariablesConstants` in the common project with constants for these environment variable names